<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>1240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54,54,54);</string>
  </property>
  <widget class="QLineEdit" name="locate_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>310</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14pt; color:rgb(255, 247, 242)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15pt; color:rgb(70, 120, 99);font-weight:600;</string>
   </property>
   <property name="text">
    <string>Locate 2:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15pt; color:rgb(70, 120, 99);font-weight:600;</string>
   </property>
   <property name="text">
    <string>Locate 1:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="locate_1">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>240</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14pt; color:rgb(255, 247, 242)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>380</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15pt; color:rgb(70, 120, 99);font-weight:600;</string>
   </property>
   <property name="text">
    <string>Locate 3:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="locate_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>380</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14pt; color:rgb(255, 247, 242)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Submit_button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>540</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);background-color: rgb(167, 168, 167);font-size:14pt;</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);font-size:28pt;</string>
   </property>
   <property name="text">
    <string>Where?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
